sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringOperations.scala

```diff
@@ -182,8 +184,9 @@ trait StringComparison extends ExpectsInputTypes {
  * A function that returns true if the string `left` contains the string `right`.
  */
 case class Contains(left: Expression, right: Expression)
-    extends BinaryExpression with Predicate with StringComparison {
+  extends BinaryExpression with Predicate with StringComparison {
```

Comment:
Why have you removed the space? See style-guide https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Indentation
I would say that `extends` should be indented like a parameter (4 spaces)